Makefile

```diff
@@ -21,7 +21,7 @@ help:
 .PHONY: dev
 dev:
 	./utils/update_nginx_conf_dev.sh
-	sudo luarocks install apisix-*.rockspec --tree deps
+	sudo luarocks install apisix-*.rockspec --tree=deps
```

Comment:
We should encourage developers who find bugs to submit PR, and we MUST thank them by adding their names.